demos/serverless/src/handlers.js

```diff
@@ -14,16 +14,44 @@ const ddb = new AWS.DynamoDB();
 // the meeting is hosted in.
 const chime = new AWS.Chime({ region: 'us-east-1' });
 
+const endpoint = process.env.CHIME_ENDPOINT;
+const globalEndpoint = "https://service.chime.aws.amazon.com";
 // Set the AWS SDK Chime endpoint. The global endpoint is https://service.chime.aws.amazon.com.
-chime.endpoint = new AWS.Endpoint(process.env.CHIME_ENDPOINT);
+chime.endpoint = new AWS.Endpoint(endpoint);
 
-const chimeRegional = new AWS.ChimeSDKMeetings({ region: 'us-east-1' });
+
+const chimeMeetingsRegionalEndPoints = getRegionalEndpoints();
+
+
+function checkValidControlRegion(controlRegion){
+  return (controlRegion == 'us-east-1' || controlRegion == 'us-west-2' || controlRegion == 'eu-central-1' || controlRegion == 'ap-southeast-1');
+}
```

Comment:
This function is duplicated in another file.  Can we put it in a common file so that this function can be de-duped?

Reply:
We can not have a common file as they have two different project folders and to have a single common file we need to set a webpack inside each of these folders.

Reply:
as for the same reason "getClientForMeeting" was dupilicated between both the files.